migrate/migrate.go

```diff
@@ -148,12 +152,44 @@ func runScripts(pool *sql.DB, scripts map[string]string, ignoreFiles []string) e
 		filenames = append(filenames, name)
 	}
 	sort.Strings(filenames)
+
 	for _, file := range filenames {
+		content, ok := scripts[file]
+		if !ok {
+			continue
+		}
+
+		var currentHash string
+		if err := pool.QueryRow("SELECT hash FROM migration_logs WHERE path = $1", file).Scan(&currentHash); err != nil && !errors.Is(err, sql.ErrNoRows) {
+			return err
+		}
+
+		hash := sha1.Sum([]byte(content))
```

Comment:
```suggestion
		hash := sha1.Sum([]byte(content))[:]
```

to avoid doing `hash[:]` below